DataForm: stop auto-injecting select placeholder (#72241)
WordPress/gutenberg: packages/dataviews/CHANGELOG.md

- DataForm control for `datetime` supports `required` and `custom` validation. [#72060](https://github.com/WordPress/gutenberg/pull/72060).

### Breaking changes

> - DataForm: Stop auto-injecting an empty "Select" placeholder into `select` controls. Consumers must now explicitly provide an empty element (for example `{ value: '', label: 'Select' }`) in `field.elements` if they want a placeholder. This is an intentional breaking change to move placeholder behavior to the component/consumer level.[#72241](https://github.com/WordPress/gutenberg/pull/#72241)

Reviewer: ```suggestion

- DataForm no longer injects an empty value into the `select` control. Consumers must now explicitly provide it if they need one. [#72241](https://github.com/WordPress/gutenberg/pull/#72241)
```

### Breaking changes

> - DataForm no longer injects an empty value to the `select` control. Consumers must now explicitly provide it if they need one. [#72241](https://github.com/WordPress/gutenberg/pull/#72241)

Reviewer: ```suggestion
- DataForm no longer injects an empty value to the `select` control. Consumers must now explicitly provide it if they need one. [#72241](https://github.com/WordPress/gutenberg/pull/72241)
```

### Breaking changes

- DataForm no longer injects an empty value into the `select` control. Consumers must now explicitly provide it if they need one. [#72241](https://github.com/WordPress/gutenberg/pull/#72241)

> 

Reviewer: ```suggestion
- DataForm no longer injects an empty value to the `select` control. Consumers must now explicitly provide it if they need one. [#72241](https://github.com/WordPress/gutenberg/pull/#72241)
```
